extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -371,37 +371,45 @@ export class AmpAnalytics extends AMP.BaseElement {
    * @private
    */
   handleEvent_(trigger, event) {
-    let request = this.requests_[trigger['request']];
-    if (!request) {
-      user().error(this.getName_(), 'Ignoring event. Request string ' +
-          'not found: ', trigger['request']);
-      return Promise.resolve();
-    }
-
-    // Add any given extraUrlParams as query string param
-    if (this.config_['extraUrlParams'] || trigger['extraUrlParams']) {
-      const params = Object.create(null);
-      Object.assign(params, this.config_['extraUrlParams'],
-          trigger['extraUrlParams']);
-      for (const k in params) {
-        params[k] = this.expandTemplate_(params[k], trigger, event);
+    const requests = Array.isArray(trigger['request'])
+        ? trigger['request'] : [trigger['request']];
+
+    const resultPromises = [];
+    for (const r in requests) {
+      let request = this.requests_[requests[r]];
+      if (!request) {
+        user().error(this.getName_(), 'Ignoring event. Request string ' +
+            'not found: ', trigger['request']);
+        return Promise.resolve();
```

Comment:
Should this instead be `resultPromises.push(Promise.resolve()); return;`? Or even w/o `push`? As it stands, one erroneous request will fail all other.